scaleway/resource_rdb_instance_test.go

```diff
@@ -118,32 +150,55 @@ func TestAccScalewayRdbInstance_Settings(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config: `
-					resource scaleway_rdb_instance main {
-						name = "test-rdb"
+					resource scaleway_vpc_private_network pn01 {
+						name = "my_private_network"
+						tags = ["tag0", "tag1", "rdb_pn"]
+					}
+				`,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("scaleway_vpc_private_network.pn01", "name", "my_private_network"),
+				),
+			},
+			{
+				Config: `
+					resource scaleway_vpc_private_network pn01 {
+						name = "my_private_network"
+						tags = ["tag0", "tag1", "rdb_pn"]
+					}
+
+					resource scaleway_rdb_instance rdbPN {
+						name = "test-rdb-with-pn"
 						node_type = "db-dev-s"
 						engine = "PostgreSQL-11"
 						is_ha_cluster = false
 						disable_backup = true
 						user_name = "my_initial_user"
 						password = "thiZ_is_v&ry_s3cret"
-						settings = {
```

Comment:
Why are the test of settings setting removed? Could it be migrated to a different test?